src/protocol/sort/reveal.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{
+    error::BoxError,
+    field::Field,
+    helpers::{
+        mesh::{Gateway, Mesh},
+        prss::SpaceIndex,
+        Direction,
+    },
+    protocol::{context::ProtocolContext, RecordId, Step},
+    secret_sharing::Replicated,
+};
+use embed_doc_image::embed_doc_image;
+use serde::{Deserialize, Serialize};
+
+/// A message sent by each helper when they've revealed their own shares
+#[derive(Debug, Serialize, Deserialize, Default)]
+pub struct AValue<F> {
```

Comment:
```suggestion
pub struct RevealValue<F> {
```

maybe?

Reply:
tbh I have no preference of one name over the other since the paper just defines this protocol in 3 lines (and doesnt have a formal definition for it to correlate). Its mostly for the infrastructure to distinguish one protocol from another.